### ISSUE TITLE
Improve cctray_xml_feed_mock

### ISSUE THREAD

**Issue by GentlemanHal** [build-canaries/nevergreen#263]
### Allow programmatic setting of CCTray XML data

This would allow tests to explicitly set the data they wished to be returned which would make them easier to read and update.

### Extract into its own package

This would make it easier for any one to use who wants to test CCTray XML feeds. For example we could use it test `clj-cctray` directly.

**Comment by joejag** [comment]
This is for the `cctray_xml_feed_mock` right?

**Comment by GentlemanHal** [comment]
Yeah, should I update the title or description to make that clearer?
